lib/deck.rb

```diff
@@ -0,0 +1,19 @@
+class Deck
+    attr_reader :cards, :category
+
+    def initialize(cards)
+        @cards = cards
+    end
+
+    def cards_in_deck
+        cards.count
+    end
+
+    def cards_in_category(category)
+        card_count = []
+        cards.each do |card|
```

Comment:
Is there another method other than `each` that could have probably been more suitable here?